packages/flutter/lib/src/material/text_selection.dart

```diff
@@ -20,64 +23,390 @@ const double _kHandleSize = 22.0;
 const double _kToolbarScreenPadding = 8.0;
 const double _kToolbarHeight = 44.0;
 // Padding when positioning toolbar below selection.
-const double _kToolbarContentDistanceBelow = 16.0;
+const double _kToolbarContentDistanceBelow = _kHandleSize - 2.0;
```

Comment:
Was 16 is now 20 - that's OK?

Reply:
I've adjusted the positioning to match native where I saw differences.  It breaks a few golden tests.